apps/src/p5lab/layoutUtils.js

```diff
@@ -0,0 +1,183 @@
+    // sprites in our group.
+    const pct = p5.constrain(count / 10, 0, 1);
+    const radius = p5.lerp(50, maxCircleRadius, pct);
+    const scale = p5.lerp(0.8, 0.3, pct * pct);
+    const startAngle = -Math.PI / 2;
+    const deltaAngle = (2 * Math.PI) / count;
+
+    group.forEach((sprite, i) => {
+      const angle = deltaAngle * i + startAngle;
+      sprite.x = 200 + radius * Math.cos(angle);
+      sprite.y = 200 + radius * Math.sin(angle);
+      sprite.rotation = (angle - startAngle) * radiansToDegrees;
+      sprite.scale = scale;
+    });
+  } else if (layout === 'grid') {
+    // Create a grid where the width is the square root of the count, rounded up,
+    // and the height is the number of rows needed to fill in count cells.
+    // For our last row, we might have empty cells in our grid (but the row
+    // structure will be the same).
+    const numCols = Math.ceil(Math.sqrt(count));
+    const numRows = Math.ceil(count / numCols);
+    group.forEach((sprite, i) => {
+      const row = Math.floor(i / numCols);
+      const col = i % numCols;
+      // || 0 so that we recover from div 0.
+      sprite.x = p5.lerp(minX, maxX, col / (numCols - 1) || 0);
+      sprite.y = p5.lerp(minY, maxY, row / (numRows - 1) || 0);
+      sprite.rotation = 0;
+    });
+  } else if (layout === 'left') {
+    createColumn(group, 100);
+  } else if (layout === 'column') {
+    createColumn(group, 200);
+  } else if (layout === 'right') {
+    createColumn(group, 300);
+  } else {
+    throw new Error('Unexpected layout: ' + layout);
+  }
+
+  // We want sprites that are lower in the canvas to show up on top of those
+  // that are higher.
+  // We also add a fractional component based on x to avoid z-fighting (except
+  // in cases where we have identical x and y).
+  group.forEach(sprite => {
+    if (p5.allSprites.indexOf(sprite) > -1) {
+      adjustSpriteDepth(sprite);
+    }
+  });
+}
+
+/**
+ * Sets the x & y positions of a group of sprites in order to arrange them
+ * in a horizontal row.
+ * @param {array<sprite>} group - The group of sprites to arrange
+ * @param {int} yLocation - The yLocation to set for all sprites
+ */
+function createRow(group, yLocation) {
+  let count = group.length;
+  for (let i = 0; i < count; i++) {
+    const sprite = group[i];
+    sprite.x = (i + 1) * (400 / (count + 1));
+    sprite.y = yLocation;
+    sprite.rotation = 0;
+  }
+}
+
+/**
+ * Sets the x & y positions of a group of sprites in order to arrange them
+ * in a vertical column.
+ * @param {array<sprite>} group - The group of sprites to arrange
+ * @param {int} xLocation - The xLocation to set for all sprites
+ */
+function createColumn(group, xLocation) {
+  let count = group.length;
+  for (let i = 0; i < count; i++) {
+    const sprite = group[i];
+    sprite.x = xLocation;
+    sprite.y = (i + 1) * (400 / (count + 1));
+    sprite.rotation = 0;
+  }
+}
+
+const adjustSpriteDepth = sprite => {
```

Comment:
After playing with this locally, I think actually be unnecessary to include it for the Sprite Lab version of layouts. In Dance Party, we magically play with the depth of sprites to give a sort of  3D foreground vs. background (smaller sprites appear to be further back). In Sprite Lab, depth logic is really only handled by the order in which the sprites are created.